sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/HiveThriftServer2.scala

```diff
@@ -35,7 +37,7 @@ import org.apache.spark.scheduler.{SparkListenerApplicationEnd, SparkListener}
  */
 object HiveThriftServer2 extends Logging {
   var LOG = LogFactory.getLog(classOf[HiveServer2])
-
+  var connectors = Seq("cassandra")
```

Comment:
@alexliu68 This can't be done with a -D system property we can read in from the connector? or if set in spark conf as 'spark.cassandra.....'? i.e. I don't think anything cassandra specific should be here.